hs/src/Reach/Simulator/Server.hs

```diff
@@ -653,6 +743,13 @@ app p srcTxt = do
     r <- caseTypes viewCall s a t
     json $ show r
 
+  post "/pass_time/:s/:n" $ do
```

Comment:
I don't know if you do this elsewhere, but you shouldn't use snake case. We use camels.